mechanicalsoup/form.py

```diff
@@ -277,6 +277,45 @@ def set(self, name, value, force=False):
             return
         raise LinkNotFoundError("No valid element named " + name)
 
+    def __getitem__(self, name):
+        """Forward arguments to :func:`~Form.get` to get value for a given form field.
+        For example, :code:`name = form["name"]` calls :code:`form.get("name")`
+        .
+        """
+        return self.get(name)
+
+    def get(self, name):
+        """Get a ``value`` from input element identified by ``name``.
+        Currently it does support "input" type only.
+
+        Example:
+
+        .. code-block:: python
+            firstname = browser['firstname']
+
+        If the element is not found , raise a :class:`LinkNotFoundError`
+        exception.
+
+"""
+        i = self.form.find("input", {"name": name})
+        if not i:
+            raise LinkNotFoundError("No input field named " + name)
+        return i["value"]
+
+    def __contains__(self, name):
+        """Test if ``name`` input field is in the form.
+        Support "input" type only.
+
+        Example:
+
+        .. code-block:: python
+           if 'name' in form:
+               ...
+
+        """
+        i = self.form.find("input", {"name": name})
```

Comment:
I was trying to think if there was any way to avoid duplication of this particular line between `__contains__` and `get`, but it seems likely that any attempt to do so would actually be more verbose. (But again this could be `inp` instead of `i`).